hs_access_control/migrations/0021_auto_20170418_2042.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
```

Comment:
@pkdash This is something I need your help.....I tried to add a new boolean field (called "approved") in ToolResource class. But migration said it is a proxy model that cannot have its own field. So I changed to add that new filed to BaseResource class. Migration worked but it also created hs_access_control/migrations/0021_auto_20170418_2042.py automatically...